code/Karma.cs

```diff
@@ -216,7 +216,7 @@ private static void OnRoundEnd( Team winningTeam, WinType winType )
 			Rebase( player );
 
 			if ( Enabled && CheckAutoKick( player ) )
-				client.Kick();
+				Game.BanPlayer( player.Client, $"Karma did not meet minimum threshold! {MinValue}", false );
```

Comment:
I think we should only ban players for low karma for one game of ttt until it changes maps.

Reply:
Yeah that was my intent, but I just realize that it doesn't work that way. I'll fix it when I add the timed bans.